src/neomaril_codex/base.py

```diff
@@ -218,23 +235,30 @@ def create_group(self, *, name: str, description: str) -> bool:
         )
 
         if response.status_code == 201:
+            t = response.json().get("Token")
             logger.info(
-                f"Group '{name}' inserted. Use the token for scoring. Carefully save it as we won't show it again."
+                f"Group '{name}' inserted. Use the token for scoring. Carefully save it as we won't show it again. Token: {t}"
             )
-            return response.json()["Token"]
-        elif response.status_code == 400:
-            logger.error(response.text)
-            logger.error("Group already exist, nothing was changed.")
-            return False
-        elif response.status_code == 401:
-            logger.error(response.text)
-            raise AuthenticationError("Login not authorized")
-        elif response.status_code >= 500:
-            logger.error(response.text)
-            raise ServerError("Server Error")
-        else:
-            logger.error(response.text)
-            raise InputError("Bad Input. Client error")
+            return HTTPStatus.OK
+
+        formatted_msg = parse_json_to_yaml(response.json())
+
+        if response.status_code == 409:
+            logger.info(f"Something went wrong:\n {formatted_msg}")
```

Comment:
Shouldn't this be a logger.error(...)?

Reply:
https://github.com/datarisk-io/mlops-neomaril-codex/pull/59/commits/193f882f1dfb5bc3b33c0a490aff890e50b90d1f